### DISCUSSION TITLE
Maintainers

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#530] [Ideas]
Maybe now that LTT has "promoted" (I put promoted in quotes because he didn't really promote it, but you get the idea), it's a good idea to get to maintainers/collaborators.
With the number of issues and PRs the flooded into the repo after the LTT video, and the fact that ThioJoe himself in the README states that "he's a total amateur" (I don't totally agree with this statement because, well you are able to write and implement new features) and that he even had to create a "Can Someone Who Knows About This Check It" label for PRs, I think maintainers/collaborators is either a REALLY good to have or a must.

I'm just throwing this out there.

**Comment by KendallDoesCoding** [comment]
> Maybe now that LTT has "promoted" (I put promoted in quotes because he didn't really promote it, but you get the idea), it's a good idea to get to maintainers/collaborators. With the number of issues and PRs the flooded into the repo after the LTT video, and the fact that ThioJoe himself in the README states that "he's a total amateur" (I don't totally agree with this statement because, well you are able to write and implement new features) and that he even had to create a "Can Someone Who Knows About This Check It" label for PRs, I think maintainers/collaborators is either a REALLY good to have or a must.
> 
> I'm just throwing this out there.

Yeah, but as you stated before @TechStudent11 its like giving someone discord mod. So, if he gives anyone maintainer or contributor, he still should tell them, dont merge anything, and he does the final approval, but for the stuff he doesnt know and adds the ""Can Someone Who Knows About This Check It" if they understand it, then ig they can merge it. Personally, I've opened a PR #521 and not yet got a response from @ThioJoe but this pr didn't have anything to do with the code so ig its not taken as priority, considering he replied to my PR relating to code, but yeah I agree with you on this.

**Comment by ThioJoe** [comment]
There's a 0% chance of this happening. I've never even hired editors for my videos. I prefer to handle everything myself.

**Reply by KendallDoesCoding** [reply]
Makes sense. That's a preference. And, you do a good job at that, considering all the issue threads and pr's in this repo.

**Reply by TechStudent10** [reply]
Understandable. Tbh I thought you had some editor for your channel.

**Reply by Firecul** [reply]
Understandable but it also adds massively to your workload. Every new project takes time away from the others.

**Reply by TechStudent10** [reply]
That is true. Now, given that Thio hasn't been active on the project for a couple of months now, I think that maintainers are now required in order to keep the project running.

**Comment by KendallDoesCoding** [comment]
same
